athena-snowflake/src/main/java/com/amazonaws/athena/connectors/snowflake/SnowflakeConstants.java

```diff
@@ -25,6 +25,8 @@
     public static final String SNOWFLAKE_NAME = "snowflake";
     public static final String SNOWFLAKE_DRIVER_CLASS = "com.snowflake.client.jdbc.SnowflakeDriver";
     public static final int SNOWFLAKE_DEFAULT_PORT = 1025;
+    public static final int PARTITION_RECORD_COUNT = 500000;
+    public static final int MAX_PARTITION_COUNT = 50;
```

Comment:
I may have asked this before, but did we try this with a higher default? It's pretty common to have more than 25M records in a table.